### PR TITLE
Revise best practices

### PR DESCRIPTION
**Pull request by redfast00**
#231

### CODE REVIEW

File: intro-HPC/ch_best_practices.tex [hunk 1]
```diff
@@ -6,81 +6,54 @@ \section{General Best Practices}
...
 
   \item  The runtime is limited by the maximum walltime of the queues. For
     longer walltimes, use checkpointing.
 
-  \item  Requesting many processors could imply long queue times.
+  \item  Requesting many processors could imply long queue times. It's adivised
+    to only request the resources you'll be able to use.
 
-  \item  For all parallel computing, request to use ``Infiniband''.
+  \item  For all multi-node jobs, please use a cluster that has ``Infiniband''.
```
Comment (boegel) [inline intro-HPC/ch_best_practices.tex]: has an Infiniband interconnect network.

File: intro-HPC/ch_best_practices.tex [hunk 2]
```diff
@@ -6,81 +6,54 @@ \section{General Best Practices}
...
 
-  \item  And above all \dots\ do not hesitate to contact the \hpc staff. We're
+  \item  And above all, do not hesitate to contact the \hpc staff. We're
     here to help you.
```
Comment (boegel) [inline intro-HPC/ch_best_practices.tex]: refer to section that explains how to contact HPC staff?

Comment (redfast00) [reply intro-HPC/ch_best_practices.tex]: Is just an emailaddress enough? The contact information is at one of the first pages, but is sitespecific (and the sites don't have a label in them yet). I can edit it, but feel like that would be out of scope for this PR.

Comment (boegel) [reply intro-HPC/ch_best_practices.tex]: Just adding the `at \hpcinfo` is enough

**Comment by boegel** [comment]
@redfast00 ready to bump version

**Pull request merged**